### PR TITLE
malone12/SVCPLAN-5322/improve_lnet_rpc_stats

### PR DESCRIPTION
**Pull request by jdmaloney**
@jakerundall Updating this repo with new Lustre module version (v3.1.0, up from 2.0.2 that was in here before)

### CODE REVIEW

**Comment by billglick** [comment]
I'm kind of torn on this request.

We know there are further fixes that need to be made to the ncsa/profile_lustre module to address stateful nodes ([SVCPLAN-5268](https://jira.ncsa.illinois.edu/browse/SVCPLAN-5268)). I think I'd prefer to hold off until that fix is in place before updating to any of the v3 ncsa/profile_lustre versions in `control-repo`.

**Comment by jakerundall** [comment]
We can update this PR, or start a new one, to reference [v3.2.0|https://github.com/ncsa/puppet-profile_lustre/releases/tag/v3.2.0] instead.

**Comment by billglick** [comment]
Can this be updated to point to tag v3.2.0?

**Comment by jdmaloney** [comment]
@billglick Yeah; will do here momentarily

**Comment by jdmaloney** [comment]
@billglick I've rebased and updated this to v3.2.0

File: Puppetfile [hunk 1]
```diff
@@ -29,7 +29,7 @@ mod 'ncsa/profile_hostbased_ssh', tag: 'v1.0.3', git: 'https://github.com/ncsa/p
 mod 'ncsa/profile_hosts', tag: 'v0.1.1', git: 'https://github.com/ncsa/puppet-profile_hosts'
 mod 'ncsa/profile_java', tag: 'v0.1.0', git: 'https://github.com/ncsa/puppet-profile_java.git'
 mod 'ncsa/profile_lmod', tag: 'v0.1.0', git: 'https://github.com/ncsa/puppet-profile_lmod.git'
-mod 'ncsa/profile_lustre', tag: 'v2.0.2',  git: 'https://github.com/ncsa/puppet-profile_lustre'
+mod 'ncsa/profile_lustre', tag: 'v3.1.0',  git: 'https://github.com/ncsa/puppet-profile_lustre'
```
Comment (billglick) [inline Puppetfile:32]: it is still showing v3.1.0.

```suggestion
mod 'ncsa/profile_lustre', tag: 'v3.2.0',  git: 'https://github.com/ncsa/puppet-profile_lustre'
```

**Review by billglick — CHANGES_REQUESTED** [review]
Still pointing v3.1..0

**Review by billglick — APPROVED** [review]

**Comment by jdmaloney** [comment]
@billglick Fixed; thought I had it in there when I rebased.

**Pull request merged**